Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -1143,9 +1139,9 @@ export function extend(publicAPI, model, initialValues = {}) {
   );
 
   model.tris = vtkHelper.newInstance();
-  model.openGLTexture = vtkOpenGLTexture.newInstance();
-  model.colorTexture = vtkOpenGLTexture.newInstance();
-  model.pwfTexture = vtkOpenGLTexture.newInstance();
+  model.openGLTexture = vtkOpenGLTexture.newInstance({ resizable: true });
```

Comment:
`ImageArrayMapper` can resize it because it deals with arbitrary stack of images. 
But I see this as a temporary fix because the `resizable` flag just bypasses the `texStorage2D` call completely and falls back to the old `texImage2D` approach entirely. What really should happen is that `vtkOpenGLTexture` should detect if there is change in size and other attributes that are immutable and delete/create the texture handle if needed. In other cases, where there is only a change in pixel data, it needs to bypass the calls to `texStorage2D` and only call `texSubImage2D`.